elastalert/config.py

```diff
@@ -238,15 +250,14 @@ def load_rules(filename, use_rule=None):
         raise EAException('Invalid time format used: %s' % (e))
 
     # Load each rule configuration file
-    rule_folder = conf['rules_folder']
-    rule_files = os.listdir(rule_folder)
     rules = []
+    rule_files = get_file_paths(conf)
     for rule_file in rule_files:
         if use_rule and rule_file != use_rule:
```

Comment:
use_rule now needs the full path -- could you lump the use_rule and file extension check into get_file_paths to reduce duplication?